nntrainer/include/nntrainer_logger.h

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (C) 2020 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * @file nntrainer_logger.h
+ * @date 02 April 2020
+ * @brief NNTrainer Logger
+ *        This allows to logging nntrainer logs.
+ * @see	https://github.com/nnstreamer/nntrainer
+ * @author Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug No known bugs except for NYI items
+ */
+
+#include <fstream>
+#include <iostream>
+#include <mutex>
+#include <string>
+#include <vector>
+
+/**
+ * @brief     Log Level of NNtrainer
+ *            0. informations
+ *            1. warnnings
+ *            2. errors
+ *            3. debugging informations
+ */
+typedef enum {
+  NNTRAINER_LOG_INFO = 0,
+  NNTRAINER_LOG_WARN,
+  NNTRAINER_LOG_ERROR,
+  NNTRAINER_LOG_DEBUG
+} nntrainer_loglevel;
```

Comment:
I think the order of log level is unusual.
usually, the log level goes this way: SILENT > FATAL > ERROR > WARN > INFO > DEBUG
I attached the log level enum values for android and tizen.
ANDROID: https://developer.android.com/reference/android/util/Log#constants_2
TIZEN:  https://docs.tizen.org/application/native/api/wearable/2.3.2/group__CAPI__SYSTEM__DLOG.html#ga87e2b0bdb074915521f34ea7e53d9c54

Reply:
Yes. I changed Level order. Thanks.